Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Audio/GetAllAudioTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.MediatR.Media.Audio.GetAll;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Xunit;
+
+using Entities = Streetcode.DAL.Entities.Media;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Audio
+{
+    public class GetAllAudioTests
+    {
+        [Fact]
+        public async Task GetAllAudio_Success()
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testAudioList = new List<Entities.Audio>()
+            {
+                new Entities.Audio() {Id = 1 }
```

Comment:
You can cut this path down using "using"

Reply:
thx, was replaced